test/common/page_objects/dashboard-page.js

```diff
@@ -21,7 +21,13 @@ const DashboardPage = function DashboardPage(browser) {
     return browser.visit(`/${accountId}?sort=${sort}`);
   };
 
+  this.filter = (accountId, filter) => {
+    assert(accountId, 'accountId is required');
+    return browser.visit(`/${accountId}?sort=updated&filter=${filter}`);
```

Comment:
name `filter(accountId, filter)` doesn't describe that it also gonna be sorted. Can we pass sort parameter to be more explicit `filter(accountId, filter, sort)` ?